Framework/Kernel/src/Matrix.cpp

```diff
@@ -1613,7 +1613,7 @@ void fillFromStream(std::istream &is, Kernel::Matrix<T> &in,
 
 // Symbol definitions for common types
 template class MANTID_KERNEL_DLL Matrix<double>;
-template class MANTID_KERNEL_DLL Matrix<int>;
+template class Matrix<int>;
```

Comment:
What's the reason that we don't have `MANTID_KERNEL_DLL` for the int definition?